judge/forms.py

```diff
@@ -150,11 +154,15 @@ def check_file(self):
     class Meta:
         model = Problem
         fields = ['is_public', 'code', 'name', 'time_limit', 'memory_limit', 'points',
-                  'statement_file', 'source', 'types', 'group', 'description']
+                  'statement_file', 'source', 'types', 'group', 'description', 'testers']
```

Comment:
- Since the org admin might not even know about the "testers" stuff. We should change the help_text to something more detailed as: "Those users can view the problem even if it is private".

I don't know whether this is possible or not, but we need this:
- If the form is using in the org view, change the "Testers" label into something like "Private users"